package.json

```diff
@@ -175,5 +178,6 @@
     "peerDependencies": {
       "@popperjs/core": "^2.11.4"
     }
-  }
+  },
+  "dependencies": {}
```

Comment:
unrelated change

Reply:
Whoop, didn't see that, sorry 😞 I'll tidy it up!